iOS.md

```diff
@@ -15,6 +15,36 @@ Refactor old sins to best of your current knowledge
   * If it isn't: Write comments for exceptional and complex code
 1. Readability trumps coolness
 
+## Spacing and Indentation
+
+* Indent using 4 spaces rather than tabs. This should be configured on the project.
+
+  ![Xcode indent settings](https://raw.githubusercontent.com/nrkno/best-practice/master/assets/xcode spacing.png)
+
+* Method braces and other braces (`if`/`else`/`switch`/`while` etc.) always open on the same line as the statement but close on a new line.
+* Don't use parentheses.
+* Tip: You can re-indent by selecting some code (or ⌘A to select all) and then Control-I (or Editor\Structure\Re-Indent in the menu).
+
+**Preferred:**
+```swift
+if video.isPlaying {
+    // Do something
+} else {
```

Comment:
> They have even had inconsistent use of tabs and spaces, used to drive me nuts.

Hah you and be both!

> Perhaps we should make templates of our own based on the style guide?

Not a bad idea :) At least if this starts to grow

If this is something you can live with I owe you 🍰 and 🍺  @BjornRuud :)



![Alt Text](https://media1.giphy.com/media/xIJLgO6rizUJi/200.gif)